README.md

```diff
@@ -39,6 +39,7 @@ scrollIntoView(someElement, {
     validTarget: function(target, parentsScrolled){
 
         // Only scroll the first two elements that don't have the class "dontScroll"
+        // Object.matches is not supported in IE11, consider using Lodash equivalent if you need to support that browser
```

Comment:
Please change this recommendation to: "consider using `matches-selector` module or equivalent if you need to support that browser"

Reply:
also change `Object.matches` -> `Element.matches`

Reply:
done!